src/Storage/AdapterPluginManager.php

```diff
@@ -22,138 +22,6 @@
-        \Zend\Cache\Storage\Adapter\Redis::class => Adapter\Redis::class,
-        \Zend\Cache\Storage\Adapter\Session::class => Adapter\Session::class,
-        \Zend\Cache\Storage\Adapter\WinCache::class => Adapter\WinCache::class,
-        \Zend\Cache\Storage\Adapter\XCache::class => Adapter\XCache::class,
-        \Zend\Cache\Storage\Adapter\ZendServerDisk::class => Adapter\ZendServerDisk::class,
-        \Zend\Cache\Storage\Adapter\ZendServerShm::class => Adapter\ZendServerShm::class,
-
-        // v2 normalized FQCNs
-        'zendcachestorageadapterapc' => Adapter\Apc::class,
-        'zendcachestorageadapterapcu' => Adapter\Apcu::class,
-        'zendcachestorageadapterblackhole' => Adapter\BlackHole::class,
-        'zendcachestorageadapterdba' => Adapter\Dba::class,
-        'zendcachestorageadapterextmongodb' => Adapter\ExtMongoDb::class,
-        'zendcachestorageadapterfilesystem' => Adapter\Filesystem::class,
-        'zendcachestorageadaptermemcache' => Adapter\Memcache::class,
-        'zendcachestorageadaptermemcached' => Adapter\Memcached::class,
-        'zendcachestorageadaptermemory' => Adapter\Memory::class,
-        'zendcachestorageadaptermongodb' => Adapter\MongoDb::class,
-        'zendcachestorageadapterredis' => Adapter\Redis::class,
-        'zendcachestorageadaptersession' => Adapter\Session::class,
-        'zendcachestorageadapterwincache' => Adapter\WinCache::class,
-        'zendcachestorageadapterxcache' => Adapter\XCache::class,
-        'zendcachestorageadapterzendserverdisk' => Adapter\ZendServerDisk::class,
-        'zendcachestorageadapterzendservershm' => Adapter\ZendServerShm::class,
-    ];
-
-    protected $factories = [
-        Adapter\Apc::class                      => InvokableFactory::class,
-        Adapter\Apcu::class                     => InvokableFactory::class,
-        Adapter\BlackHole::class                => InvokableFactory::class,
-        Adapter\Dba::class                      => InvokableFactory::class,
-        Adapter\ExtMongoDb::class               => InvokableFactory::class,
-        Adapter\Filesystem::class               => InvokableFactory::class,
-        Adapter\Memcache::class                 => InvokableFactory::class,
-        Adapter\Memcached::class                => InvokableFactory::class,
-        Adapter\Memory::class                   => InvokableFactory::class,
-        Adapter\MongoDb::class                  => InvokableFactory::class,
-        Adapter\Redis::class                    => InvokableFactory::class,
-        Adapter\Session::class                  => InvokableFactory::class,
-        Adapter\WinCache::class                 => InvokableFactory::class,
-        Adapter\XCache::class                   => InvokableFactory::class,
-        Adapter\ZendServerDisk::class           => InvokableFactory::class,
-        Adapter\ZendServerShm::class            => InvokableFactory::class,
```

Comment:
Aren't these still needed?

Reply:
theyre provided  by the satellites. 
v3 of this package will use dependency inversion and thus required any package which will implement the `laminas/laminas-cache-storage-adapter-implementation`.